app/js/question_controller.js

```diff
@@ -14,17 +14,38 @@
   var question_object;
   var solution_id;
 
+  var TOPICS = {
+    '': 'Choose a topic',
+    'email': 'Email',
+    'clock': 'Clock/Alarm',
+    'camera': 'Camera/Gallery/Video',
+    'audio': 'Audio/Music/RadioFM',
+    'performance': 'Performance',
+    'radios': 'Celular Data/WiFi/Bluetooth',
+    'sms': 'SMS/MMS',
+    'calls': 'Calling&Contacts',
+    'browser': 'Browser/Download',
+    'storage': 'Storage/SD card',
+    'geolocation': 'HERE Maps/GPS',
+    'keyboard': 'Keyboard',
+    'system-updates': 'System Updates',
+    'language': 'Language',
+    'marketplace': 'Marketplace/Apps Install',
+    'other': 'Other'
+  };
+  var TOPIC_PREFIX = 'topic:';
```

Comment:
We shouldn't have `TOPIC_PREFIX` in two places. Having this live only in sumo_db makes sense I believe. `SumoDB.get_question` should return an object that hides the implemenation detail of how the category is stored. aka `question_object.category`.

Let's name everything category btw, to not confuse with the topic field already stored in the DB.